egs/chime4/asr1_mc/local

```diff
@@ -0,0 +1 @@
+../asr1/local
```

Comment:
better to use own local directory (not symlink), but use a symlink for each file.
But if you don't use any local files for this recipe, this solution is OK.

Reply:
Currently, I do not use any local file.